.circleci/config.yml

```diff
@@ -325,6 +325,23 @@ steps-release-build: &steps-release-build
     - *step-maybe-notify-slack-failure
     - *step-maybe-notify-slack-success
 
+steps-native-mksnapshot-build: &steps-native-mksnapshot-build
+  steps:
+    - attach_workspace:
+        at: .
+    - *step-depot-tools-add-to-path
+    - *step-setup-env-for-build
+
```

Comment:
This build step needs a gn gen:
https://github.com/electron/electron/blob/master/.circleci/config.yml#L294..L297